daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -1841,3 +1841,31 @@ class RazerViperV3ProWireless(RazerViperV3ProWired):
     METHODS = RazerViperV3ProWired.METHODS + ['set_hyperpolling_wireless_dongle_indicator_led_mode']
 
     POLL_RATES = [125, 500, 1000, 2000, 4000, 8000]
+
+
+class RazerDeathAdderV3HyperSpeedWired(__RazerDevice):
+    """
+    Class for the Razer DeathAdder V3 HyperSpeed (Wired)
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*usb-Razer_Razer_DeathAdder_V3_HyperSpeed_000000000000-if0(1|2)-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x00C4
+    METHODS = ['get_device_type_mouse', 'max_dpi', 'get_dpi_xy', 'set_dpi_xy', 'get_dpi_stages', 'set_dpi_stages',
+               'get_poll_rate', 'set_poll_rate',
+               # Battery
+               'get_battery', 'is_charging', 'get_idle_time', 'set_idle_time', 'get_low_battery_threshold', 'set_low_battery_threshold']
+
+    DEVICE_IMAGE = "https://dl.razerzone.com/src/6124/6124-1-en-v2.png"
+
+    DPI_MAX = 26000
+
+
+class RazerDeathAdderV3HyperSpeedWireless(RazerDeathAdderV3HyperSpeedWired):
+    """
+    Class for the Razer DeathAdder V3 HyperSpeed (Wireless)
+    """
+
+    USB_PID = 0x00C5
+
+    POLL_RATES = [125, 500, 1000]
```

Comment:
This is the default, why are you setting it explicitly for the Wireless variant? Or are you missing to set it in the Wired variant?

Reply:
`POLL_RATES = [125, 500, 1000]` is default, so you can just remove it if this device doesn't have poll rate > 1000Hz support or some other weird limitation.